src/lib/components/tippies/profile_dropdown.svelte

```diff
@@ -50,6 +52,8 @@
             }
         }
     };
+
+    beforeUpdate(() => {});
```

Comment:
umm whats with this empty code?

Reply:
https://github.com/tokitou-san/CoreProject-V3-UI/assets/61817579/2164d1ff-68b2-4db3-bce2-e48382c5c39a

Reply:
I am not merging until work is finished